packages/block-editor/src/components/inspector-advanced-controls/index.native.js

```diff
@@ -0,0 +1 @@
+export default () => null;
```

Comment:
I'm checking the `inspector-advanced-controls/index.js` file, it seems to define the SlotFill pattern for Inspector Advanced Controls. The web definition should work just fine, and we might need it once we actually implement advanced controls on mobile 🤔